ocaml/xapi/create_misc.ml

```diff
@@ -411,6 +434,8 @@ let make_software_version ~__context =
 		"xencenter_min", Xapi_globs.xencenter_min_verstring;
 		"xencenter_max", Xapi_globs.xencenter_max_verstring;
 		"network_backend", Network_interface.string_of_kind (Net.Bridge.get_kind dbg ());
+		"xen-livepatches", get_xen_livepatch_list ();
```

Comment:
I'm really sorry, I should have said this earlier, but I notice that the other software version fields use underscore as a separator. So it should be "xen_livepatches", "kernel_livepatches".